gems/pending/VMwareWebService/MiqPbmInventory.rb

```diff
@@ -0,0 +1,33 @@
+require 'VMwareWebService/PbmService'
+
+module MiqPbmInventory
```

Comment:
Interested in your opinion on that as well.  What I didn't want to do was clutter the `MiqVim` class with pbm methods, but also wanted the pbm methods to be callable directly from a `MiqVim` instance like the `MiqVimInventory` methods are.  Those work because `MiqVim` inherits from `MiqVimInventory` so this was as close as I thought but if there's a better way to organize I'm definitely open to it.